packages/rich-text-editor/src/styles/vaadin-rich-text-editor-base-styles.js

```diff
@@ -0,0 +1,352 @@
+  }
+
+  [part~='toolbar-button-color']::before {
+    mask-image: var(--_vaadin-icon-color);
+  }
+
+  [part~='toolbar-button-color']::after {
+    background-color: var(--_color-value, currentColor);
+  }
+
+  [part~='toolbar-button-background']::before {
+    mask-image: var(--_vaadin-icon-background);
+  }
+
+  [part~='toolbar-button-background']::after {
+    background-color: var(--_background-value, currentColor);
+  }
+
+  [part~='toolbar-button-color']::after,
+  [part~='toolbar-button-background']::after {
+    bottom: 50%;
+    content: '';
+    display: block;
+    height: var(--vaadin-icon-size, 1lh);
+    mask-image: var(--_vaadin-icon-color-underline);
+    position: absolute;
+    transform: translateY(50%);
+    width: var(--vaadin-icon-size, 1lh);
+  }
+
+  [part~='toolbar-button-h1']::before {
+    mask-image: var(--_vaadin-icon-h1);
+  }
+
+  [part~='toolbar-button-h2']::before {
+    mask-image: var(--_vaadin-icon-h2);
+  }
+
+  [part~='toolbar-button-h3']::before {
+    mask-image: var(--_vaadin-icon-h3);
+  }
+
+  [part~='toolbar-button-subscript']::before {
+    mask-image: var(--_vaadin-icon-subscript);
+  }
+
+  [part~='toolbar-button-superscript']::before {
+    mask-image: var(--_vaadin-icon-superscript);
+  }
+
+  [part~='toolbar-button-list-ordered']::before {
+    mask-image: var(--_vaadin-icon-list-number);
+  }
+
+  [part~='toolbar-button-list-bullet']::before {
+    mask-image: var(--_vaadin-icon-list-bullet);
+  }
+
+  [part~='toolbar-button-align-left']::before {
+    mask-image: var(--_vaadin-icon-align-left);
+  }
+
+  [part~='toolbar-button-align-center']::before {
+    mask-image: var(--_vaadin-icon-align-center);
+  }
+
+  [part~='toolbar-button-align-right']::before {
+    mask-image: var(--_vaadin-icon-align-right);
+  }
+
+  [part~='toolbar-button-image']::before {
+    mask-image: var(--_vaadin-icon-image);
+  }
+
+  [part~='toolbar-button-link']::before {
+    mask-image: var(--_vaadin-icon-link);
+  }
+
+  [part~='toolbar-button-blockquote']::before {
+    mask-image: var(--_vaadin-icon-quote);
+  }
+
+  [part~='toolbar-button-code-block']::before {
+    mask-image: var(--_vaadin-icon-code);
+  }
+
+  [part~='toolbar-button-clean']::before {
+    mask-image: var(--_vaadin-icon-clear);
+  }
+
+  @media (forced-colors: active) {
+    [part~='toolbar-button']:focus,
+    [part~='toolbar-button']:hover {
+      outline: 1px solid !important;
+    }
+
+    [part~='toolbar-button'][on] {
+      outline: 2px solid;
+      outline-offset: -1px;
+    }
+  }
+`;
+
+const states = css`
+  :host([readonly]) [part='toolbar'] {
+    display: none;
+  }
+
+  :host([disabled]) {
+    pointer-events: none;
+    opacity: 0.5;
+    -webkit-user-select: none;
+    user-select: none;
+  }
+
+  :host([disabled]) [part~='toolbar-button'] {
+    cursor: var(--vaadin-disabled-cursor);
+    opacity: 0.5;
+    pointer-events: var(--_vaadin-button-disabled-pointer-events, none);
```

Comment:
Does this apply to RTE toolbar buttons?

Reply:
Ported over from `vaadin-rich-text-editor-core-styles.js`:
```
:host([disabled]) [part~='toolbar-button'] {
  background-color: transparent;
}
```

Just updated to match the base disabled button styles.

Reply:
@vursen, does the "disabled button accessibility" feature cover RTE toolbar buttons?

Reply:
> does the "disabled button accessibility" feature cover RTE toolbar buttons?

No, because they are currently native HTML buttons.

Reply:
Thanks. Then we can remove this property.